### ISSUE TITLE
Multi-page conditional active tab is hidden when it is the active page

### ISSUE THREAD

**Issue by romainpoirier** [verbb/formie#522]
**Description**
The multi-page conditional active tab is hidden when it is the active page.

**Steps to reproduce**
1. Setup a simple multi-pages form ;
2. On each page, add any field like a _Single-line Text_ field ;
3. Setup a basic conditional rule on the second page to show it if it matches the field value of the first page ;
4. On front-end using `craft.formie.renderForm`, type the matching value on the field of the first page ;
5. After typing, the second tab will show up when it get the match ;
6. On submit we are correctly redirected to page 2, but while on it its tab is hidden (`data-conditionally-hidden="true"` remains) ;

### Form settings
- Multi-page form: Yes
- Submission Method: Page Reload
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: dev-craft-3 as 1.4.9
- Craft version: 3.7.0
- Multi-site: No

**Comment by engram-design** [comment]
This is (almost) a duplicate of https://github.com/verbb/formie/issues/521 which is the same issue. Whether it be page conditions or field conditions, they're treated largely the same for page reload based forms where all pages aren't loaded in.

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.12",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.13](https://github.com/verbb/formie/releases/tag/1.4.13)
